build-uboot-standalone.sh

```
@@ -0,0 +1,175 @@
+RKBIN_REPO="https://github.com/rockchip-linux/rkbin.git"
+WORK_DIR="${PWD}/uboot-build-standalone"
+OUTPUT_DIR="${WORK_DIR}/output"
+BOARD="nanopi_zero2"
+
+# ARM64 cross-compiler - try Yocto SDK first, then system
+YOCTO_BUILD="${PWD}/build"
+SDK_PATH="${YOCTO_BUILD}/tmp/deploy/sdk"
+
+# Check for installed Yocto SDK
+if [ -d "/opt/poky" ]; then
+    ENV_SETUP=$(ls /opt/poky/environment-setup-* 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)